2016/cities.html

```diff
@@ -0,0 +1,147 @@
+    </div>
+    <div class="column-one-half">
+      {% include tweet.html user_display_name="Tim Cowland" user_handle="Tim_SBIG" tweet_html='Amazing service from <a href="https://twitter.com/Herts_Highways">@Herts_Highways</a>. Faulty street lamp reported via <a href="https://twitter.com/fixmystreet">@fixmystreet</a> fixed with 34 mins of sending message. <a href="https://twitter.com/hashtag/speedoflight?src=hash">#speedoflight</a>' tweet_url="https://twitter.com/Tim_SBIG/status/725256139051536384" date="27 Apr 2016"  %}
+    </div>
+  </div>
+  <div class="row">
+    <div class="column-one-half">
+      {% include tweet.html user_display_name="Eadlin" user_handle="DeeHants" tweet_html='<a href="https://twitter.com/mysociety">@mysociety</a>, your <a href="https://twitter.com/fixmystreet">@fixmystreet</a> app makes it so much easier to report issues. I found another 3 on today’s drive home!' tweet_url="https://twitter.com/DeeHants/status/722131033253089280" date="18 Apr 2016"  %}
+    </div>
+    <div class="column-one-half">
+      {% include tweet.html user_display_name="Plexus Design" user_handle="plexusdesigner" tweet_html='<a href="https://fixmystreet.com">fixmystreet.com</a> 3 min form has fixed an issue that has been bugging me and my neighbours for months! Thank you! <a href="https://twitter.com/mysociety">@mysociety</a>' tweet_url="https://twitter.com/plexusdesigner/status/709519365603991552" date="14 Mar 2016"  %}
+    </div>
+  </div>
+</div>
+
+<div class="section section--white section--cities-councils">
+  <h1>Huge improvements on FixMyStreet for Councils…</h1>
+  <p>FixMyStreet has been totally transformed for some of our most important users: the council staff who receive reports. It’s all thanks to our collaboration with some of the UK’s most forward-thinking councils.</p>
+  <p>Working with <strong>Oxfordshire County Council</strong>, we’ve <a href="https://www.google.com/url?q=https://www.fixmystreet.com/about/council&sa=D&ust=1481717542577000&usg=AFQjCNFo7BqUVyBlHNA9ljbGruZZXcJzag">dramatically improved the feature set of FixMyStreet for Councils</a>, allowing council workers to track and deal with reports more efficiently.</p>
+  <p>And <strong>Bristol Council</strong> have led the way in showing how to replace outdated and expensive suppliers, while still retaining future flexibility, <a href="https://www.mysociety.org/2016/11/22/something-in-the-middle-how-bristol-connects/">through the power of middleware</a>.</p>
+</div>
+
+<div class="section section--white section--flipped">
+  <h1>…And for our community around the world</h1>
+  <p>Since FixMyStreet is an open source project, many of the improvements we’ve made for our UK councils—like <a href="http://fixmystreet.org/2016/11/29/v2.0-html-emails/">HTML emails</a> and <a href="http://fixmystreet.org/2015/10/23/v1.7/">easy report filtering</a>—will also benefit our international re-users. It’s win-win!</p>
+</div>
+
```

Comment:
I know that Prostate Cancer and POD aren't exactly 'better cities' but this is the only page I can fit them on if we don't do a Commercial section, so how about a section:

<H1>Technology for better health</H1>

We used our coding skills for good, on two very different projects.
[split columns:]
<h2>The Patent Opposition Database</H2>
Working with Médecins Sans Frontières, we helped those <a href="https://www.mysociety.org/2016/03/09/patent-oppositions-re-igniting-a-campaign-for-fairer-access-to-drugs/">opposing medicine pricing monopolies</a>. 

<h2>Campaigning for Erectile Dysfunction care</H2>
And, with Prostate Cancer UK, we used FOI and boundary-mapping to create <a href="https://bettercare.prostatecanceruk.org/">a smooth campaigning tool</a>.

Also: there's supposed to be a blog post about the PCUK tool, I'll get Jen to chase them and then maybe we can link to that instead of directly to the tool.